actix-http/src/config.rs

```diff
@@ -323,13 +323,14 @@ mod tests {
         settings.set_date(&mut buf1);
         let now1 = settings.now();
 
-        sleep_until(Instant::now() + Duration::from_secs(1)).await;
+        sleep_until(Instant::now() + Duration::from_secs(2)).await;
+        yield_now().await;
```

Comment:
required to let the bg task update clock due to single threaded test runtime? do i have that right?

Reply:
I find ci handles async delay poorly. 
I just hope a 2 second delay and a yield would actually make the date service timer updated for once.